jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/LifeCycleBaseTest.java

```diff
@@ -2388,7 +2390,9 @@ public void beforeTaskStartedEvent(TaskEvent event) {
 
             @Override
             public void beforeTaskCompletedEvent(TaskEvent event) {
-                assertNull(event.getTask().getTaskData().getTaskInputVariables());                
+                assertNotNull(event.getTask().getTaskData().getTaskInputVariables());
+                assertEquals(1, event.getTask().getTaskData().getTaskInputVariables().size());
+                assertTrue(event.getTask().getTaskData().getTaskInputVariables().containsKey("input"));                
```

Comment:
what I meant is that having empty task input/output data was not guaranteed as there might be multiple listeners registered, and once one of them loads the data it will be then non empty for all subsequent listeners